src/nvim/CMakeLists.txt

```diff
@@ -339,4 +345,81 @@ add_library(nvim-test MODULE EXCLUDE_FROM_ALL ${NEOVIM_GENERATED_SOURCES}
 target_link_libraries(nvim-test ${NVIM_LINK_LIBRARIES})
 set_property(TARGET nvim-test APPEND_STRING PROPERTY COMPILE_FLAGS -DUNIT_TESTING)
 
+# Helper targets that verify that specific headers can be included alone (i.e.
+# do not depend on other includes)
+set(HEADER_CHECK_FILES)
+set(NO_SINGLE_CHECK_HEADERS
```

Comment:
consider renaming this to `HEADER_CHECK_SKIPPED` for parallel form with `HEADER_CHECK_TARGETS` et al.